test/new-e2e/utils/e2e/client/vm_client.go

```diff
@@ -35,7 +35,11 @@ func newVMClient(t *testing.T, sshKey string, connection *utils.Connection) (*vm
 
 // ExecuteWithError executes a command and returns an error if any.
 func (vmClient *vmClient) ExecuteWithError(command string) (string, error) {
-	return clients.ExecuteCommand(vmClient.client, command)
+	output, err := clients.ExecuteCommand(vmClient.client, command)
+	if err != nil {
+		return "", fmt.Errorf("%v: %v", output, err)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Why there is need for formatting output and error ? Can output have content on error ? I would have expected it to be empty

Reply:
`ExecuteCommand` uses `CombinedOutput` which include stderr. `err` contains no information about the error except the exit code which is not very useful.

Reply:
Ah ! Thank you !